docs/release-notes/8.0.asciidoc

```diff
@@ -113,6 +113,11 @@ A new Lucene 9 validation change may cause event correlation (EQL) rule errors w
 . Refresh the page to verify the deprecation was resolved, then return to the guided steps on the Upgrade Assistant page.
 // end::breaking-changes[]
 
+[discrete]
+[[deprecations-8.0.0]]
+==== Deprecations
+* The `output_index` parameter is no longer supported for the APIs that create and update rules. In addition, the `output_index` field is not included in the exported rule file.
```

Comment:
`the `output_index` field is not included in the exported rule file.` 
Unfortunately, this part is not yet fixed, I created this bug ticket https://github.com/elastic/kibana/issues/155355 to be fixed by the team. Until then the `output_index` will still be part of the exported rule but it doesn't have any impact

Reply:
Gotcha, thank you! I'll remove it from the release notes summary then. Would you recommend any other changes to the first sentence or does it accurately describe the deprecated field?

```suggestion
* The `output_index` parameter is no longer supported for the APIs that create and update rules. 
```

Reply:
Thank you Nastasha! All is good 👍